hack/gen-bootstrap-bindata.sh

```diff
@@ -17,6 +17,7 @@ pushd vendor/github.com/jteeuwen/go-bindata > /dev/null
 popd > /dev/null
 
 pushd "${OS_ROOT}" > /dev/null
+GOPATH=$(os::build::get-go-bin-location-from-gopath go-bindata)
   "${GOPATH}/bin/go-bindata" -nocompress -nometadata -prefix "bootstrap" -pkg "bootstrap" \
```

Comment:
I think you'd want to use the output of your function here instead of `"${GOPATH}/bin/go-bindata"`, instead of over-writing `$GOPATH`